scripts/docker-as-symfony

```diff
@@ -0,0 +1,593 @@
+: ${DATABASE_URL:="mysql://root:root@db/symfony"}
+: ${DATABASE_DUMP:="/var/backups/db-reset.sql"}
+: ${SMTP_HOST:="mail:1025"}
+: ${REQUIRED_DIRECTORIES:=""}
+: ${LOCK_TIME:=""}
+: ${TEST_SERVER_PORT:="8888"}
+
+# Script variables
+DATABASE_HOST=$(a=${DATABASE_URL#*@}; echo ${a%/*})
+DATABASE_NAME=${DATABASE_URL##*/}
+DATABASE_USER=$(a=${DATABASE_URL#*//}; echo ${a%:*})
+DATABASE_PASSWORD=$(a=${DATABASE_URL#*//}; b=${a#*:}; echo ${b%@*})
+SYMFONY_IS_INSTALLED=0
+SCRIPT_NAME=$(basename $0)
+
+# Print help
+function print_help() {
+  echo "usage: $SCRIPT_NAME <action> [options]
+
+  * bootstrap
+
+    Install Symfony and dependencies. Require config export, do not bootstrap a new projet.
+
+    --skip-dependencies      # Do not run composer and yarn install
+    --skip-install           # Do not run Drupal install (only if arealdy installed)
+    --skip-db-reset          # Do not reset database using SQL dump (only when --skip-install)
+    --install-only           # Same as --skip-dependencies --skip-styleguide-build
+
+  * setup
+
+    Ensure Symfony is porper setup (parameters.yml, mail settings, ...).
+
+    --with-dependencies      # Run composer and yarn install
+
+  * db-reset
+
+    Reset database using SQL dump made after bootstrap.
+
+    --update-dump            # Update database dump (include updated Drupal config, but not
+                             # default content)
+
+  * db-dump
+
+    Create a database dump that can be used as with db-restore (not with db-reset).
+
+    --file=<path>            # Path to dump file (not required)
+
+  * db-restore
+
+    Restore database from dump created with db-dump.
+
+    --file=<path>            # Path to dump file (not required)
+
+  * apache-server
+
+    Run web server using 'apache'. Reset database if not present.
+
+    --with-db-reset          # Reset database before launch server
+    --cache                  # Do not invalidate opcache (server must be restarted
+                             # reload cache)
+    --help                   # Display apache help
+    -- <...>                 # any apache valid args
+
+  * phpunit
+
+    Run phpunits tests.
+
+    --skip-default-stops     # Do not stop on error and failure (remove --stop-on-error --stop-on-failure)
+    --with-dependencies      # Run composer and yarn install
+    --with-db-reset          # Reset database before launch server
+    --group=<group>          # Only runs tests from the specified group(s)
+    --exclude-group=<group>  # Exclude tests from the specified group(s)
+    --help                   # Display phpunit help
+    -- <...>                 # any phpunit valid args
+
+  * quality-check
+
+  Run code quality tools (phpcs).
+
+    --skip-phpcs             # Do not run phpcs
+    --skip-phpcpd            # Do not run phpcpd
+    --fail-on-phpcs          # Command fail if phpcs fail too
+    --fail-on-phpcpd         # Command fail if phpcpd fail too
+  "
+}
+
+# Setup
+function setup() {
+  return;
+}
```

Comment:
can we remove the setup step ? We don't need it thanks to the ENV parameters.

Reply:
Yes we can, was a place holder but how you say we not need it

Reply:
done.